config/skim_UL2016_mib.cfg

```diff
@@ -99,59 +99,67 @@ skipTriggers = false
 #variables  = VBFjj_dEtaSign:BDT_j1j2n_dEtaSign, HH_deltaR:BDT_n_HH_deltaR, dib_dEtaSign:BDT_n_b1b2_dEtaSign, VBFjj_deltaEta:BDT_j1j2n_dEta, VBFjj_mass:BDT_j1j2n_Mjj, VBFjet2_pt:BDT_j2n_pt, tauH_pt:BDT_n_ditau_pt, bH_VBF1_deltaEta:BDT_bbH_j1n_dEta, VBFjet1_btag:BDT_j1n_csv, VBFjet1_pt:BDT_j1n_pt, dib_deltaR:BDT_n_dib_deltaR, VBFjet2_PUjetID:BDT_j2n_PUjetID
 variables  = VBFjet1_pt:BDT_j1n_pt, bH_pt:BDT_n_bH_pt, dib_dEtaSign:BDT_n_b1b2_dEtaSign, ditau_deltaR:ditau_deltaR, HH_deltaR:BDT_n_HH_deltaR, HH_zV:BDT_HH_zV_n, VBFjj_dEtaSign:BDT_j1j2n_dEtaSign, VBFjj_mass:BDT_j1j2n_Mjj
 
 [DNN]
-computeMVA = false
-weights  = /gwpool/users/dzuolo/HHbbtautauAnalysisUL/CMSSW_11_1_0_pre6/src/cms_runII_dnn_models/models/nonres_gluglu/2020-07-31-0/ensemble
-features = /gwpool/users/dzuolo/HHbbtautauAnalysisUL/CMSSW_11_1_0_pre6/src/cms_runII_dnn_models/models/nonres_gluglu/2020-07-31-0/features.txt
+computeMVA = true
+weights  = /gwpool/users/spalluotto/HH_bbtautau/CMSSW_11_1_9/src/cms_runII_dnn_models/models/res_gluglu/2020-08-03-0/ensemble
+features = /gwpool/users/spalluotto/HH_bbtautau/CMSSW_11_1_9/src/cms_runII_dnn_models/models/res_gluglu/2020-08-03-0/features.txt
 kl = 1
 
 [HHbtag]
-weights = /gwpool/users/dzuolo/HHbbtautauAnalysisUL/CMSSW_11_1_0_pre6/src/HHTools/HHbtag/models/HHbtag_v1_par_
+weights = /gwpool/users/spalluotto/HH_bbtautau/CMSSW_11_1_9/src/HHTools/HHbtag/models/HHbtag_v1_par_
 
 [HHReweight]
-inputFile = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/allHHnodeMap_5DdiffRew_2016.root
+inputFile = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_9/src/KLUBAnalysis/weights/allHHnodeMap_5DdiffRew_2016.root
 histoName = allHHNodeMap
-coeffFileNLO = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/coefficients_pm_pw_NLO_Ais_13TeV_V2.txt
-coeffFileLO = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_0_pre6/src/KLUBAnalysis/weights/coefficients_pm_pw_LO_Ais_13TeV_V2.txt
+coeffFileNLO = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_9/src/KLUBAnalysis/weights/coefficients_pm_pw_NLO_Ais_13TeV_V2.txt
+coeffFileLO = /gwpool/users/brivio/Hhh_1718/LegacyRun2/May2020/LIMdev/CMSSW_11_1_9/src/KLUBAnalysis/weights/coefficients_pm_pw_LO_Ais_13TeV_V2.txt
 
 [Multiclass]
 computeMVA = false
 
+[diTauMLMass]
+computeMVA = true
+weightsLO = /gwpool/users/brivio/Hhh_1718/add_DiTau_ML/CMSSW_11_1_9/src/KLUBAnalysis/weights/diTauMLMassModels/modelLO/v1
+weightsLE = /gwpool/users/brivio/Hhh_1718/add_DiTau_ML/CMSSW_11_1_9/src/KLUBAnalysis/weights/diTauMLMassModels/modelLE/v1
+weightsHO = /gwpool/users/brivio/Hhh_1718/add_DiTau_ML/CMSSW_11_1_9/src/KLUBAnalysis/weights/diTauMLMassModels/modelHO/v1
+weightsHE = /gwpool/users/brivio/Hhh_1718/add_DiTau_ML/CMSSW_11_1_9/src/KLUBAnalysis/weights/diTauMLMassModels/modelHE/v1
+
+
```

Comment:
I don't remember seeing this. Is it needed?

Reply:
Actually we can remove it. I added these lines to the config file while I was testing the HTT mass regression developed by Simone.